docker-compose.yaml

```diff
@@ -363,6 +363,15 @@ services:
     restart: always
     volumes: ["./docker/karton.ini:/etc/karton/karton.ini", "${DOCKER_COMPOSE_ADDITIONAL_SHARED_DIRECTORY:-./shared}:/shared/"]
 
+  karton-sql_injection_detector:
+    <<: *artemis-build-or-image
+    command: "python3 -m artemis.modules.sql_injection_detector"
+    depends_on: [karton-system, karton-logger]
+    env_file: .env
+    restart: always
+    volumes: ["./docker/karton.ini:/etc/karton/karton.ini", "${DOCKER_COMPOSE_ADDITIONAL_SHARED_DIRECTORY:-./shared}:/shared/", "./:/opt"]
```

Comment:
don't mount ./ in prod

Reply:
Yea, its only for me, I have to make new module for develop version for this tip.

Reply:
```python
    def process(self, data: Any) -> Any:
        services_to_create_volume = ["web", "karton-sql_injection_detector"]

        for service in data["services"]:
            if service in services_to_create_volume and "./:/opt" not in data["services"][service]["volumes"]:
                data["services"][service]["volumes"].append("./:/opt")
        return data
```
Maybe like that

Reply:
yes, but looping over all kartons